Doc/library/calendar.rst

```diff
@@ -365,7 +396,7 @@ For simple text calendars this module provides the following functions.
 
 .. function:: month(theyear, themonth, w=0, l=0)
 
-   Returns a month's calendar in a multi-line string using the :meth:`~TextCalendar.formatmonth`
+   Returns a month's calendar in a multi-line :class:`str` using the :meth:`~TextCalendar.formatmonth`
```

Comment:
The original formulation was also fine with me. In general we try to keep code changes minimal, so reverting this is also an option. (similar remark for the other changes like this)

Reply:
I agree on reverting this.